feat: Allow traefik merged dynamic configuration override with <project>.yaml (#6375)
ddev/ddev: pkg/ddevapp/dotddev_assets/traefik/README.md

#ddev-generated

The project `traefik` directory usually contains only DDEV-generated files.

The `certs` directory contains default certs, and the `config` directory normally contains only
default dynamic configuration in the `config/<project>.yaml` file.

Additional dynamic configuration can be merged into the DDEV-generated `<project>.yaml` by
adding `dynamic_config.*.yaml` files to the `traefik` directory. As with the global `static_config.*.yaml` files,
merging is done with an **override** strategy, meaning that the final file in alphanumeric sort to touch a
particular element of the YAML structure wins.

> The `dynamic_config.middleware.yaml.example` can be used as a starting point for adding

Reviewer: ```suggestion
The `dynamic_config.middlewares.yaml.example` can be used as a starting point for adding
```

> There is a `dynamic_config.middleware.yaml.example` file that you can use as a starting point for adding

Reviewer 2: ```suggestion
The `dynamic_config.middleware.yaml.example` can be used as a starting point for adding
```
